lib/util/git/conflicts-cache.ts

```diff
@@ -33,11 +33,13 @@ export function setCachedConflictResult(
   isConflicted: boolean
 ): void {
   const cache = getCache();
-  cache.gitConflicts ??= {};
-  const { gitConflicts } = cache;
+  const { gitConflicts = {} } = cache;
```

Comment:
If cache object is empty, we'll never store `gitConflicts` in it, isn't it?

Reply:
Not sure. The cache will be empty first time when branch is created but we will want to store isConflicted -> false .